corehq/apps/app_manager/management/commands/migrate_localized_media.py

```diff
@@ -40,6 +40,7 @@ def migrate_app(self, app_doc):
                 'task_list',
                 'referral_list',
                 'case_list_form',
+                'search_config',  # add this?
```

Comment:
This command looks pretty old and not referenced, is it possible it could just be deleted? Maybe @sravfeyn could advise?